packages/@sfx/sayt-plugin/tsconfig.esnext.json

```diff
@@ -18,7 +18,7 @@
     "sourceMap": true,
     "allowSyntheticDefaultImports": true,
     "noImplicitAny": true,
-    "allowJs": true
```

Comment:
@jmbtutor Just want to verify that this repo didn't need this for any testing? I know we needed it for the view layer.